src/routers/course.ts

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
```

Comment:
Is this work you actually did?

Reply:
Could be a git issue? It doesn't say I modified anything on my end - this was Seiya's work